app.models/articles.models.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection");
+
+exports.fetchArticleById = (article_id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: `No article found for article: ${article_id}`,
+        });
+      } else {
+        return rows;
+      }
+    });
+};
```

Comment:
Good work on this:

- Again, `fetchArticleById` is a good function name
- SQL statement is good, and well done using the parameterised query functionality to avoid SQL injection
- You've implemeneted the 404 not found well by checking if the array is empty, well done